libafl_sugar/src/forkserver.rs

```diff
@@ -137,7 +137,7 @@ impl<'a, const MAP_SIZE: usize> ForkserverBytesCoverageSugar<'a, MAP_SIZE> {
             );
 
             // A feedback to choose if an input is a solution or not
-            let objective = feedback_or_fast!(CrashFeedback::new(), TimeoutFeedback::new());
+            let objective = feedback_or_fast!(CrashFeedback::new(), TimeoutFeedback::new(), None);
```

Comment:
I'm not sure why we want to add a `None` here?

Reply:
It seems I followed the "if it compiles, it's good" philosophy here :) I'll take a closer look and run some tests, thanks for the feedback!